File: administrative_duties/events/dtos/event_dto.py
class EventDto:
    client_id: str
    event_id: str
    location_id: str
    organizer_id: str
    name: str
    category: str
    description: str
    day_of_week: int
    start_time: str
    duration_time: str
    cost: int
    is_active: bool

    def __init__(self, **kwargs):
        self.__dict__.update(kwargs)

    def __getitem__(self, item):
        return getattr(self, item)

File: app/src/main/java/com/ebsolutions/dal/dtos/EventDto.java
package com.ebsolutions.dal.dtos;

import io.micronaut.serde.annotation.Serdeable;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.NonNull;
import lombok.experimental.SuperBuilder;
import lombok.extern.slf4j.Slf4j;
import software.amazon.awssdk.enhanced.dynamodb.mapper.annotations.DynamoDbBean;

import java.time.LocalTime;

@Data
@DynamoDbBean
@Serdeable
@Slf4j
@SuperBuilder
@AllArgsConstructor
@NoArgsConstructor
public class EventDto extends DatabaseDto {
    @NonNull
    private String locationId;
    @NonNull
    private String organizerId;
    @NonNull
    private String name;
    private String category;
    @NonNull
    private String description;
    private int dayOfWeek;
    private LocalTime startTime;
    private short duration;
}
